index.js

```diff
@@ -10,16 +10,13 @@ function glslifyWebpackLoader(source) {
   this.callback = this.async()
   this.cacheable(true)
 
-  glslify.bundle(source, {
-    inline: true,
-    basedir: basedir
-  }, function(err, src, files) {
-    if (files) {
-      files.forEach(function(file) {
-        self.addDependency(file)
-      })
-    }
-
-    return self.callback(err, src)
-  })
+  try {
+    const comp = glslify(source, {
+      inline: true,
+      basedir: basedir
+    });
```

Comment:
Is looks like `glslify` is now synchronous? It appears this is probably as it needs to be; just raising the potential concern due to the change.

Reply:
There's no reference to a callback anymore, so yeah